src/ResourceManager/Dns/Commands.Dns/Commands.Dns.csproj

```diff
@@ -57,6 +57,9 @@
     <Compile Include="Models\DnsBaseCmdlet.cs" />
```

Comment:
Need to also update the reference in Commands.Dns.Netcore

Reply:
updated. thanks